src/styles/index.scss

```diff
@@ -1,3 +1,170 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+  color: #060b35;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  margin: 0;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__picture {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: 16px;
+
+    .image {
+      width: 160px;
+      height: 134px;
+      margin: 0 auto;
+    }
+
+    .title {
+      display: flex;
+      flex-direction: column;
+      margin-top: 40px;
+
+      &-name {
+        font-size: 12px;
+        font-weight: 500;
+        line-height: 18px;
+        margin: 0 0 4px;
+      }
+
+      &-description {
+        font-size: 10px;
+        font-weight: 400;
+        line-height: 14px;
+        color: #616070;
+        margin: 0;
+      }
+    }
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .content {
```

Comment:
You should have only 2 BEM blocks
![image](https://github.com/mate-academy/layout_product-cards/assets/64101898/eb18590b-a48b-45f8-96fb-87bd382abd57)